bin/game.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Game
+  def initialize(board)
+    @game_board = board
+
+    @game_board.ask_names
+    @game_board.draw
+  end
+
+  def start
+    while @game_board.has_movs
+      @game_board.mark
+      @game_board.draw
+    end
+
+    if @game_board.winner
+      puts "#{@game_board.winner.name} WINS!!"
+    else
+      puts 'This is a draw'
```

Comment:
**[REQUIRED]**
* All the user interaction code should reside in the `main.rb` file as specified [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/).

Reply:
moved to main.rb
```

  if my_board.winner
    puts "#{my_board.winner.name} WINS!!"
  else
    puts 'This is a draw'
  end

Reply:
added a option to play again
```
  loop do
    puts 'Would you like to play again ? y/n '
    rp = gets.chomp

    if %w[y n].include?(rp)
      play_again = rp == 'y'
      break
    end
  end